README.md

```diff
@@ -1 +1,71 @@
-# spark-integration-hub-k8s-operator
+# Spark Integration Hub K8s Operator
+
+The Spark Configuration Hub charm, is the charm that enables the integration of 
+the Charmed Spark solution with existing charms (PostgreSQL, S3-integrator, etc) 
+and bundles (COS, Identity, etc) and properly configure the corresponding 
+option in the Spark ecosystem.
+
+The main tasks of the configuration hub charm are the following:
+
+* Handle relations with other charms and translate them in configuration options 
+  that Spark can handle (like the Spark History Server charm does with the S3-integrator).
+* Store the configuration options in Kubernetes secrets for the service account
+  managed by the Charmed Spark solution (all service account are labeled with predefined labels)
+* Handle the addition/removal of service accounts with corresponding updates of 
+  configurations.
+* Handle updates from the related charms (i.e., new database credentials, 
+  change of endpoints, updated S3 credentials.)
+* The configuration hub charm may also be able to create and configure services
```

Comment:
This feature is not available yet

Reply:
But we provide the interface already [here](https://github.com/canonical/spark-integration-hub-k8s-operator/blob/main/metadata.yaml#L42), don't we?